vllm/worker/model_runner.py

```diff
@@ -819,8 +840,7 @@ def capture_model(self, kv_caches: List[torch.Tensor]) -> None:
                     context_lens=context_lens[:batch_size],
                     block_tables=block_tables[:batch_size],
                     use_cuda_graph=True,
-                    kv_cache_dtype=self.kv_cache_dtype,
-                )
+                    kv_cache_dtype=self.kv_cache_dtype)
```

Comment:
nit: What is this change for?

Reply:
Hmm not sure why it is formatted. Reverted